lxd/instance/drivers/driver_qemu.go

```diff
@@ -2153,6 +2153,11 @@ func (d *qemu) generateConfigShare() error {
 		return err
 	}
 
+	err = shared.FileCopy(shared.VarPath("server.crt"), filepath.Join(configDrivePath, "lxd.crt"))
```

Comment:
We shouldn't pass this as a file but instead provide it through the same API call which passes the host address and port.

The reason for that is that a server certificate will change during instance live migration, so we need the receiving host to be able to provide the new connection informations including the expected server certificate.